Client/Frontend/Autofill/CreditCard/CreditCardSettingsView/CreditCardInputViewModel.swift

```diff
@@ -140,8 +163,7 @@ class CreditCardInputViewModel: ObservableObject {
             alertBody: Text(CreditCardText.RemoveCardSublabel),
             primaryButtonStyleAndText: .destructive(Text(CreditCardText.RemovedCardLabel)) { [self] in
                 guard let creditCard = creditCard else { return }
```

Comment:
This now gives a warning since `let creditCard` isn't used anymore

Reply:
Good catch, my XCode was acting with warnings. Pushing a fix